tests/scripts/multi_chip/run_end_to_end_demos.sh

```diff
@@ -16,4 +16,8 @@ fi
 cd $TT_METAL_HOME
 export PYTHONPATH=$TT_METAL_HOME
 
-# Your tests go here!
+# Falcon40B prefill 60 layer end to end with 10 loops; we need 8x8 grid size
+WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/t3000/falcon40b/tests/ci/test_falcon_end_to_end_60_layer_t3000_prefill_10_loops.py
```

Comment:
Should we export this an env var in the workflow?

cc: @tapspatel

Reply:
Don't think this should be enabled by default. This is a feature in metal to put dispatch on ethernet cores, and its up to devs to enable or not.